h2o-core/src/main/java/hex/DistributionFactory.java

```diff
@@ -389,7 +389,7 @@ public double deviance(double w, double y, double f) {
         if (Math.abs(y - f) <= _huberDelta) {
             return w * (y - f) * (y - f); // same as wMSE
         } else {
-            return 2 * w * (Math.abs(y - f) - _huberDelta) * _huberDelta; // note quite the same as wMAE
+            return 2 * w * (Math.abs(y - f) - _huberDelta/2) * _huberDelta; // note quite the same as wMAE
```

Comment:
We can avoid division if rewrite to:
```
w * (2 * Math.abs(y - f) - _huberDelta) * _huberDelta
```

Reply:
Aaa, now I can see the bug more clear. The bug was not missing division by 2, but incorrect multiplication of formula in the bracket by 2. I think somebody wants to avoid division and rewrite the formula incorrectly.

So you are right @michalkurka! The correct and optimal formula is 

```suggestion
            returnw * (2 * Math.abs(y - f) - _huberDelta) * _huberDelta; // note quite the same as wMAE
```